src/sidebar/components/test/button-test.js

```diff
@@ -115,28 +109,24 @@ describe('Button', () => {
     assert.calledOnce(fakeOnClick);
   });
 
-  it('adds additional class name passed in `className` prop', () => {
-    const wrapper = createComponent({ className: 'my-class' });
-
-    assert.isTrue(wrapper.hasClass('my-class'));
-  });
-
-  it('sets compact style if `useCompactStyle` is set`', () => {
-    const wrapper = createComponent({ useCompactStyle: true });
+  it('applies icon-only classname if no visible label present', () => {
```

Comment:
both "class name" and "classname" both appear in this file. I suggest just calling it "class"